frontend/src/components/Comment/CommentList.js

```diff
@@ -1,43 +1,36 @@
 import React from 'react'
-import Comment from './Comment'
-import { getCommentByPicture } from './CommentAPI';
+import { useState, useEffect } from 'react'
+import { getCommentByPicture } from './CommentAPI'
+import {Comment} from './Comment'
+import './SCSS/CommentList.scss'
 
-class CommentList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            commentList: []
-        }
-    }
-    componentDidMount() {
-        this.loadComments();
-    }
+const CommentList = (props) => {
 
-    componentDidUpdate(prevProps, prevState, snapshot) {
-        if(prevProps.postId !== this.props.postId) {
-            this.loadComments();
-        }
-    }
+    const [commentList, setcommentList] = useState([])
+
+    useEffect(() => {
+        loadComments();
+    }, [props.postId, props.refreshComment]);
 
-    loadComments() {
-        getCommentByPicture(this.props.postId).then((response) => {
-            this.setState({commentList: response});
+    const loadComments = () => {
+        console.log(props);
+        getCommentByPicture(props.postId).then((response) => {
+            console.log(response);
+            setcommentList(response);
         });
     }
 
-    render() {
-        return (
+    return (
+        <div>
```

Comment:
Not sure what this div does considering you already have `comment-list-wrapper` as your encompassing/wrapper div?